ducanhpham0312/zeevision
# feat: jobs table for /instance/id page and JobsPage

### Linked issue

Closes ducanhpham0312/zeevision-private#7
Closes ducanhpham0312/zeevision-private#116
Closes ducanhpham0312/zeevision-private#64


### Description

Backend:
- Add InstanceKey field for Job
- Add jobs to be queryable (needed for JobsPage)
- Update sample query
Frontend:
- Add jobs table to JobsPage and SingleInstancePage
- Use Tab from @mui/base to switch between tables in SingleInstancePage. Needs more styling

### Testing

- Run test bench  and the application
- Open pgAdmin and use the updated query in `fill_db.sql`.
- Open Playground. Both queries below should return corresponding jobs.
```
### Query all jobs. With the sample query, it should return 4 jobs.
{
  jobs {
    instanceKey
    key
    type
    retries
    state
    time
  }
}


### Query jobs that belong to an instance. The following query should return 2 jobs.
{
  instance(instanceKey: 11) {
    jobs {
      key
      type
    }
  }
}
```
- Open the application. The `/jobs` page should include a table with 4 jobs. Each job includes the instance key (which looks clickable), click it will open the corresponding instance. 
- The `/instances/11` page should have a tab to switch between variables and jobs. Clicking Jobs should show a table with 2 jobs.
